comment_spell_check.py

```diff
@@ -377,6 +377,11 @@ def add_dict(enchant_dict, filename, verbose=False):
 
     # You better not have more than 1 word in a line
     for wrd in lines:
+        if not wrd.replace("'", "").isidentifier():
+            print(
+                "Warning: adding word with non-alphanumeric characters to dictionary:",
+                wrd,
+            )
         if not enchant_dict.check(wrd):
```

Comment:
Thanks for addressing this issue. Just for my understanding, does it still make sense to add words that contain _numeric_ characters to the dictionary? For example, "image1", "image2", "image3", etc?  Because I had the impression that a dictionary entry should only have _alphabetic_ characters!

 - #54